### ISSUE TITLE
Cannot call plugin multiple times anymore

### ISSUE THREAD

**Issue by rlex** [dj-wasabi/ansible-telegraf#39]
After PR #37, i cannot use multiple instances of same plugin.
Example:
```yaml
telegraf_plugins_extra: 
 - plugin: ping
    config:
      - urls = [
                "host1.int",
                "host2.int"
               ]
      - count = 4
      - ping_interval = 1.0
      - timeout = 1.0
  - plugin: ping
     config:
       - urls = [
                "host1",
                "host2"
               ]
      - count = 4
      - ping_interval = 1.0
      - timeout = 1.0
```

Role tries to write first instance, then second, creating always-changed state.

**Comment by dj-wasabi** [comment]
Hi @rlex 

Can you check with the version available on github? #40 is merged and might help you?

**Comment by rlex** [comment]
Yep, works with filename param. Thanks.
